components/raftstore/src/store/config.rs

```diff
@@ -299,6 +299,10 @@ pub struct Config {
     #[doc(hidden)]
     pub long_uncommitted_base_threshold: ReadableDuration,
 
+    /// Max time for the entry cache to be warmed up.
```

Comment:
minor: Max duration  for warming up entry cache. The original comment can be misunderstood as the max time (count) a specific entry cache can be warmed up.

Reply:
fixed